docs/example-scenario/ai/newsfeed-ingestion.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Mass ingestion and analysis of news feeds on Azure
+description: Create a pipeline for ingesting and analyzing text, images, sentiment, and other data from RSS news feeds using only Azure services, including Azure Cosmos DB and Azure Cognitive Services.
+author: njray
+ms.date: 2/1/2019
+ms.custom: azcat-ai, AI
+social_image_url: /azure/architecture/example-scenario/ai/media/mass-ingestion-newsfeeds-architecture.png
+ms.topic: example-scenario
+ms.service: architecture-center
+ms.subservice: example-scenario
+---
+
+# Mass ingestion and analysis of news feeds on Azure
+
+This example scenario describes a pipeline for mass ingestion of electronic documents using public RSS news feeds. It then provides near-real-time analysis using Azure Cognitive Services to offer useful insights as the documents are placed in durable storage.
+
+A generator is used to populate Azure Cosmos DB with RSS articles in multiple languages, and then trigger several actions. These actions demonstrate the use of available Cognitive Services, including text translation, facial recognition, and detecting sentiment.
+
+This scenario is based on [English][english], [Russian][russian], and [German][german] news feeds, but you can easily extend it to other RSS feeds. For ease of deployment, the data collection, processing, and analysis are based entirely on Azure services.
+
+## Relevant use cases
+
+Other relevant use cases include:
+
+* Translate any text to the language of choice.
+* Find key phrases, entities, and user sentiment in digital content.
+* Detect objects, text, and landmarks in images associated with a digital article.
+* Detect people by their gender and age in any image associated with digital content.
+
+## Architecture
+
+![][architecture]
+
+The data flows through the solution as follows:
+
+1. An RSS news feed acts as the generator that obtains data from a document or article. For example, with an article, data typically includes a title, a summary of the original body of the news item, and sometimes images
+
+2. A generator or ingestion process inserts the article and any associated images into an Azure Cosmos DB [Collection][collection].
+
+3. A notification triggers an Ingest function in Azure Functions that stores the image content, if any, in an Azure Storage account as blobs. The article is then passed to the next queue.
```

Comment:
The article text is stored in CosmosDB and the article images (if any) are stored in Blob Storage. 

I think Ingest was capitalized because that is the actual name of the Azure Function, but agree that maybe it doesn't need capitalization.